Dockerfile

```diff
@@ -38,6 +38,16 @@ RUN zypper addrepo https://download.opensuse.org/repositories/isv:SUSE:Edge:Edge
     nm-configurator && \
     zypper clean -a
 
+# Make adjustments for running guestfish and image modifications on aarch64
+# guestfish looks for very specific locations on the filesystem for UEFI firmware
+# and also expects the boot kernel to be a portable executable (PE), not ELF.
+RUN zypper install -y qemu-uefi-aarch64 && \
```

Comment:
I'd suggest to include the package in the zypper installation layer above. This way we can simplify this to only make the file changes.